pandunia-loge.csv

```diff
@@ -1116,7 +1116,7 @@ fenkate||||analysis (dissection)|analyse|análisis (disección)|análise|ана
 fenu||||split (fall apart)||partir (dividirse)||||||||||||||jakautua (hajota)|pęknąć (rozpaść się)
 fere|mate|Fe|spa:hierro, por:ferro, fra:fer, may:ferum|iron||hierro|||||||||||||fero|rauta|żelazo
 fermium|mate|Fm||fermium||fermio|||||||||||||fermio|fermium|ferm
-festa||||celebrate (party)||celebrar (festejar)|||||||||||||festi|juhlia|świętować, imprezować
+fa feste||||celebrate (party)||celebrar (festejar)|||||||||||||festi|juhlia|świętować, imprezować
 feste|||por:festa, spa:fiesta, fra:fête, deu:Fest, eng:festival, rus:фестиваль (festival')|party (celebration, festival)||celebración (fiesta)|||||||||||||festo|juhlat|przyjęcie, święto, festiwal
```

Comment:
**festa** is better. It is also supported by Malay "pesta" (borrowed from Portuguese).